test/lib/cli/commands/build.js

```diff
@@ -81,7 +81,7 @@ test("ui5 build self-contained", async (t) => {
 	await build.handler(args);
 	t.deepEqual(
 		builderStub.getCall(0).args[0],
-		Object.assign({}, {selfContained: true}, defaultBuilderArgs),
+		Object.assign({}, defaultBuilderArgs, {selfContained: true}),
```

Comment:
Object.assign is shallow. However, defaultBuilderArgs is deep.

I'd suggest creating a helper function that always returns a new object.

Reply:
Oh right! I misread that.